cse-runner-app/src/test/java/com/farao_community/farao/cse/runner/app/dichotomy/DichotomyRunnerTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ *  * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ *  * This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.farao_community.farao.cse.runner.app.dichotomy;
+
+import com.farao_community.farao.cse.computation.BorderExchanges;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+class DichotomyRunnerTest {
+
+    @Test
+    void testSplittingFactorsConversion() {
+        Map<String, Double> splittingFactors = Map.of(
+            "AT", 0.1,
+            "CH", 0.2,
+            "FR", 0.3,
+            "SI", 0.4
+        );
+        Map<String, Double> convertedSplittingFactors = DichotomyRunner.convertSplittingFactors(splittingFactors);
+        assertEquals(0.1, convertedSplittingFactors.get(CseCountry.AT.getEiCode()), 0.01);
```

Comment:
I think it's better to declare the value of the delta as a constant since we use the same value several times.

Reply:
Done